sympy/vector/coordsysrect.py

```diff
@@ -98,6 +98,33 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
```

Comment:
I'm not sure why do that.
You assume that we have to use `transformation` or (`rotation` and `translation`), so `transformation` should have information about every three transformations.
I think that it could be easier sometimes to split transformation into steps. For example for `transformation` set `cylindrical` and for location set some `vector`.